pytorch_lightning/trainer/__init__.py

```diff
@@ -1095,4 +1095,3 @@ def tbptt_split_batch(self, batch, split_size):
 from pytorch_lightning.utilities.seed import seed_everything
 
 __all__ = ["Trainer", "seed_everything"]
-
```

Comment:
can you remove all these whitespace changes so we get fewer files changed?

Reply:
sure, just wanted to see all greens in tests locally. I'll remove them.